Sources/SnapshotTesting/Internal.swift

```diff
@@ -3,7 +3,7 @@ import Cocoa
 typealias Image = NSImage
 typealias ImageView = NSImageView
 typealias View = NSView
-#elseif os(iOS) || os(tvOS) || os(watchOS)
```

Comment:
watchOS doesn't include XCTest or unit testing in general, so no need to support!